contributivity.py

```diff
@@ -82,7 +82,7 @@ def __str__(self):
         output += "Computation time: " + computation_time + "\n"
         output += (
             "Number of characteristic function computed: "
-            + str(self.charac_fct_calls_count)
+            + str(self.first_charac_fct_calls_count)
```

Comment:
Why first ?

Reply:
Because we count only the first time we compute the characteristic function. The second time we call the characteristic function on the same subset is not relevant as it is a simple call to the memory.